content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Datadog FIPS Compliance
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "agent/proxy"
+  tag: "Documentation"
+  text: "Agent Proxy Configuration"
+algolia:
+  rank: 80
+  tags: ["fips", "fips proxy", "compliance"]
+---
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+<div class="alert alert-warning">The Datadog FIPS proxy is only available in the US1-FED region.</a></div>
```

Comment:
Can we also flag that this is Public Beta state to set expectations with customers. Thanks!